CP2/cp2.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Check Point 2</title>
+  <link rel="stylesheet" href="cp2.css">
+</head>
+
+
+<div class="parent">
```

Comment:
This div is outside of the body tag.  All visible content should be inside the <body></body> tags